util/k8sutil/k8sutil.go

```diff
@@ -121,3 +122,19 @@ func FilterPodCount(pods []*v1.Pod, phase v1.PodPhase) int32 {
 	}
 	return result
 }
+
+func GetTotalReplicas(replicas map[commonv1.ReplicaType]*commonv1.ReplicaSpec) int32 {
+	tfjobReplicas := int32(0)
```

Comment:
Remove mentions of "tfjob" in this function

Reply:
done